recipes/napari-plugin-manager/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "napari-plugin-manager" %}
+{% set version = "0.1.0a2" %}
```

Comment:
Just a note this is ok to do as there has [never been a non-alpha release]( https://pypi.org/project/napari-plugin-manager/#history )

Whenever there is a formal release, new alphas, etc. will then need to go under the appropriate label

Here's [the relevant text from CFEP 5]( https://github.com/conda-forge/cfep/blob/31efda3f858fe3574db0418b9d4b4b907159f6f0/cfep-05.md?plain=1#L93-L101 ):

> ## Exceptions
>
> Certain packages (for example [black](https://pypi.org/project/black/#history)) follow
a release cycle in which they have never had a non-beta/alpha release.  In these cases
the conda packages for those do *not* need to be published to a prerelease label.
>
> Once a non-prerelease version of such a package is available and has been merged into
the associated feedstock, they lose this exception and should publish *future* prereleases
to a prerelease label like other packages.